packages/api-clients/open-api/authorizationApi.yml

```diff
@@ -2232,6 +2263,29 @@ components:
         - createdAt
         - description
         - keys
+    PartialProducerKeychain:
+      type: object
+      description: |
+        Models a producer keychain with a reduced set of fields: only the fields that are
+        visible to any caller tenant.
+      additionalProperties: false
+      properties:
+        visibility:
+          type: string
+          enum: [PARTIAL]
+        id:
+          type: string
+          format: uuid
+        producerId:
+          description: |
```

Comment:
As for client, add this description also under `producerId` in `FullProducerKeychain`